src/components/dashboard/main-content/course-enrollments/CourseEnrollments.jsx

```diff
@@ -110,6 +110,13 @@ const CourseEnrollments = ({ children }) => {
     );
   }
 
+   const handleOnClose = () => {
+    // 1) setShowCancelledAssignmentsAlert(false);
+    // 2) patchLearnerContentAssignment
```

Comment:
There may be more than 1 canceled assignment at the time "Dismiss" is clicked, so this might be updated to `patchLearnerContentAssignments`. The intent is that when "Dismiss" is clicked, we should attempt to dismiss (i.e., `has_learner_dismissed=True`) for _all_ canceled assignments in a single API call.

The `patchLearnerContentAssignment` function currently assumes working with an individual assignment at a time, which could be a frontend performance / UX concern if we were to make an API call for each assignment (say the learner has 5-10 canceled assignments). 

Ideally, clicking "Dismiss" would result in a single API call being made to dismiss all canceled assignments for the learner/customer pair.